src/components/allcluster/allcluster.js

```diff
@@ -77,7 +79,7 @@ const AllCluster = () => {
     setIsModalVisible(false);
     // console.log(name, database, version);
     createNotification("success", "Redirecting!");
-    var answer = await handleOk(name, database, version);
+    var answer = await handleOk(name, database, version,username,password);
```

Comment:
The `handleOk` function is defined here -- https://github.com/spinup-host/spinup-dash/blob/11fc7144bbe9a72876f8ff1e51fecc128c4c8476/src/api/createService.js#L4

You should update it to accept username and password as parameters (and send it as part of the request).